pkg/client/client.go

```diff
@@ -30,19 +31,29 @@ func (n Node) Loggable() map[string]interface{} {
 func (n Node) Bootstrap(ctx context.Context) error {
 	// TODO:  update this when we replace 'ps' with a
 	//        capability set.
-	return n.ps.Client.Resolve(ctx)
+	if err := n.ps.Client.Resolve(ctx); err != nil {
+		return err
+	}
+	return n.view.Client.Resolve(ctx)
 }
 
 // Done returns a read-only channel that is closed when
 // 'n' becomes disconnected from the cluster.
 func (n Node) Done() <-chan struct{} {
-	return n.conn.Done()
+	return n.conns[0].Done()
```

Comment:
I guess checking one of the connections is sufficient to know if the client was (partially) disconnected, but we could also check all of them